src/App.jsx

```diff
@@ -0,0 +1,48 @@
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import About from "./pages/About";
+import HomePage from "./pages/HomePage";
+import Navigation from "./components/Navigation";
+
+import "./server";
+import Footer from "./components/Footer";
+import VansList from "./pages/VansList";
+import VanDetails from "./pages/VanDetails";
+import Income from "./pages/Host/Income";
+import Reviews from "./pages/Host/Reviews";
+import HostLayout from "./components/Layouts/HostLayout";
+import Dashboard from "./pages/Host/Dashboard";
+import { Vans as HostVans } from "./pages/Host/Vans";
+import { VanDetails as HostVandetails } from "./pages/Host/VanDetails";
+import { SingleVanDetails as Details } from "./components/vans/SingleVanDetails";
+import { Pricing } from "./components/vans/Pricing";
+import Photos from "./components/vans/Photos";
+
+function App() {
+    return (
+        <BrowserRouter>
+            <Navigation />
```

Comment:
Could you move the navigation inside a layout route?

Reply:
It seems hard for me. I will do it in the finish ups.